packages/wrangler/src/index.ts

```diff
@@ -400,6 +402,13 @@ export function createCLIParser(argv: string[]) {
 			type: "string",
 			requiresArg: true,
 		})
+		.option("env-file", {
+			describe:
+				"Path to an .env file to load - can be specified multiple times - earlier ones are overridden by later ones",
```

Comment:
"earlier ones are overridden by later ones"

Maybe clarify that the values are overridden, not the files